__tests__/circulars.ts

```diff
@@ -466,6 +508,50 @@ describe('parseEventFromSubject', () => {
     expect(parseEventFromSubject(baksanSubjectWithHyphen)).toBe(baksanEvent)
   })
   //#endregion
+
+  //#region XRF
```

Comment:
I haven't noticed these `#region` comments before, but I guess they have been here for a while. Can we just put these into jest `describe` blocks rather than relying on magic comments?

Reply:
Sure.